### ISSUE TITLE
training_special_tokens skip loss

### ISSUE THREAD

**Issue by Hanzhang-lang** [AkariAsai/self-rag#95]
https://github.com/AkariAsai/self-rag/blob/1fcdc420e48f50a7d7ab1ece5494221b93252e99/data_creation/train_special_tokens.py#L213-L224
If multiple paragraph references(\<paragraph\>) appear here, will all the content in between be set to IGNORE_INDEX? Is this incorrect?

**Comment by knowledgetamer** [comment]
yes i find this isue too,maybe fix this can raise the perfomance alot

**Comment by knowledgetamer** [comment]
a break is preferred
